django-stubs/utils/timezone.pyi

```diff
@@ -30,10 +32,10 @@ class LocalTimezone(ReferenceLocalTimezone):
 
 utc: UTC = ...
 
-def get_fixed_timezone(offset: Union[timedelta, int]) -> tzinfo: ...
-def get_default_timezone() -> tzinfo: ...
+def get_fixed_timezone(offset: Union[timedelta, int]) -> FixedOffset: ...
```

Comment:
The [Django code](https://github.com/django/django/blob/187a64608d2a207dfc12fdd6816526e9550069b7/django/utils/timezone.py#L28) doesn't return a FixedOffset, it returns a `datetime.timezone`. So I'd either change to `datetime.timezone` or keep it as `tzinfo`.

Reply:
Ok. It was changed in Django 2.2. I looked into wrong version of the codebase.